log4j-core-test/src/main/java/org/apache/logging/log4j/core/test/junit/LoggerContextSource.java

```diff
@@ -84,20 +80,4 @@
      * Specifies the time unit {@link #timeout()} is measured in.
      */
     TimeUnit unit() default TimeUnit.SECONDS;
-
-    /**
-     * Toggles Log4j1 configuration file compatibility mode for XML and properties files.
-     */
-    boolean v1config() default false;
-
-    /**
-     * Determines whether to bootstrap a fresh LoggerContextFactory.
-     */
-    boolean bootstrap() default false;
```

Comment:
No longer relevant; we just bootstrap them all the time depending on where the annotation is used.